dna/examine.go

```diff
@@ -55,6 +55,13 @@ func CountGaps(seq []Base) int {
 	return gapCount
 }
 
+// GCContent returns the GC content for the input sequence. Note that n/Ns are ignored.
```

Comment:
maybe write something in the comment that it truncates to nearest int

Reply:
Good idea